simphony_metadata.yml

```diff
@@ -0,0 +1,398 @@
+VERSION: 1.0
+
+CUDS: Common Universal Data Structure
+
+Purpose: CUDS provides a semantic layer for the SimPhoNy metadata schema 
+
+Resources:
+ CUDS 1.0 (1st Edition): http://simphony.eu/cuds/1.0/ # does not exist
+ CUDS Issues Page: https://github.com/simphony/simphony-metadata/issues
+ CUDS Mailing List: simphony-ssb@fraunhofer.de
+ CUDS Reference Parser: https://github.com/simphony/simphony-metadata/cparser #?
+
+# Any all caps entity that has a "CUBA." prefix is assumed to be
+# defined and it is being used.  Any all caps entity without the
+# "CUBA." prefix, is being defined.
+
+# The PARENT relation is an explicit inheritance of data! It defines
+# specific first order relations between data entities.  parent is
+# used in the same context as type in OO programming, however, parent
+# indicates a directional first order relation.
+
+# CUBA.KEY (KEY can be any data key) are attributes intrinsic to the
+# data entity defined.
+
+# Each CUBA.KEY has its own definition as an entity, which defines its
+# type (parent) in a unique universal manner.
+
+# Each CUBA.KEY can have additional implementation-related metadata,
+# these metadata are the following:
+
+# default: a default value, when absent, no default is implied and the
+# user has to eventually supply a value (not necessarily upon
+# instantiation, implementation dependent).
+ 
+# If omitted, no default is assumed, and the user has to supply a
+# value.
+
+# scope: CUBA.USER or CUBA.SYSTEM.  if omitted, the CUBA.USER is
+# assumed by default
+
+# shape: as [i,j,] specifying that it is a sequence of entities of its
+# own type.  [i, j] means an array of i*j types.  [i, ] means the
+# sequence or array can have minimum of i elements. [, i] the array can
+# have any, maximum j elements. If omitted, a shape of [1] is assumed.
```

Comment:
Thanks for noting the typo. 

The notation for the list may not be the best, this should be seen as an intermediate solution. There is a need to define values for some entities (such as CUBA.PHYSICS_VARIABLE) that are lists/sequence/series of a CUBA. We try to avoid creating a list type for each type by providing the option of shape, similar to what Ioanis introduced in cuba.yml. I first used a notation with a comma, but I found it later to be a bit inconsistent with standard notation, so I changed it to use colons, however, the push was not complete! here is how the comment should look like (will be pushed shortly: 

# shape: [i_min:i_max] specifies that the value is a sequence of entities of its own type.
# In each dimension, the first index is the minimum while the second maximum number of items allowed.
# similarly:
# shape: [i_min:i_max,j_min:j_max] specifies that the value is a multidimensional 
# sequence of entities of its own type.  In each dimension, the first index
# is the minimum while the second maximum number of items allowed.
# 

Basically, we need a way to convey the information that the value is a list/array/sequence. This may still not be the best solution. Any ideas to enhance this are welcome.

# shape: [i_min:i_max,j_min:j_max] specifying that it is a multidimensional 
# sequence of entities of its own type.  In each dimension, the first index
# is the minimum while the second maximum number of items allowed.